### ISSUE TITLE
Cart is not recovered after sending an reminder email in WC 2.6.14

### ISSUE THREAD

**Issue by Chetna1510** [TycheSoftwares/Woocommerce-abandon-cart-lite#164]
I have abandoned a cart as a logged-in user on woo_aclite and then I have received an abandoned cart reminder email for my cart. I login to account (Not from Email) and I have placed an order via PayPal. Then I have checked Recovered Orders tab, my cart is not displaying under Recovered Orders tab.

My Abandoned Cart ID is : #168

**Comment by Chetna1510** [comment]
With WooCommerce 2.6.14, I am unable to replicate the issue. Screenshot: https://www.screencast.com/t/UZwNUUK489

I have abandoned a cart and placed an order via PayPal using the link from abandoned cart reminder email. My cart is being disaplyed on the Recovered Orders Tab.
Screenshot of Order Received page: https://www.screencast.com/t/5v0FOFaqusKn
Screenshot: https://www.screencast.com/t/CqL0Kh0M

So, I am closing this issue.

**Comment by bhavik-kiri** [comment]
I have git hub copy ( 30-01-2018 )
I have tried to replicate the issue and results are as follows.


For the different browser, it has updated the status of the cart. I have abandoned the cart from chrome then I have waited for the reminder email. Then once I have received the email. I have cleared the cache of the Firefox and logged in to the site and placed the order for the same cart.

1. Before placing: https://prnt.sc/i7o81q
2. After placing: https://prnt.sc/i7o8c0
3. Recover tab: https://prnt.sc/i7o8tp

When I have abandoned the cart in the FireFox and waited for the reminder email (Until that time I have closed the firefox ). Once I have received the reminder email I have logged in to site in the same browser without clearing the cache. 

And I have directly placed the order. Then it has considered that order as the recovered order.

**So, we need to discuss that If we should consider the cart as record irrespective of the browsers if the reminder email is sent to that cart**.
